src/tests/tests/storage.js

```diff
@@ -164,6 +164,8 @@
   });
 
   QUnit.test("DNT does not cascade", (assert) => {
+    let settings_map = storage.getBadgerStorageObject('settings_map');
+    settings_map.setItem("checkForDNTPolicy", true);
```

Comment:
Good point, but that's a bug with our testing setup. Let's fix or open a ticket to fix this unit tests bug instead of resetting settings in every test that uses them.